app/uk/gov/hmrc/eoricommoncomponent/frontend/connector/SUB09SubscriptionDisplayConnector.scala

```diff
@@ -45,9 +45,9 @@ class SUB09SubscriptionDisplayConnector @Inject() (http: HttpClient, appConfig:
   private val logger = Logger(this.getClass)
   private val url    = appConfig.getServiceUrl("subscription-display")
 
-  def subscriptionDisplay(
-    sub09Request: Seq[(String, String)]
-  )(implicit hc: HeaderCarrier): Future[Either[EoriHttpResponse, SubscriptionDisplayResponse]] = {
+  def subscriptionDisplay(sub09Request: Seq[(String, String)], originatingService: String)(implicit
```

Comment:
can originatingService be implicit here as well?

Reply:
As discussed,marking as implicit makes the code more complex